### PR TITLE
Movies database seeding

### LINKED ISSUE #2: Seed Movies Data

### PR DESCRIPTION
**Pull request by aleksakojadinovic**
Closes #2 

Implemented seeding for the movies database, the process is explained in the readme.
Note that I've moved the changes that implement controllers, repositories etc to a different branch (still local) because this branch was specifically made for seeding. It will be done as a separate issue.

The seeding is done using an sql script and csv files that get mounted into the docker container. Also I've added a data volume so that the data is not lost when the container gets removed.

### CODE REVIEW

File: Services/Movies/Movies.API/dbseed.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿docker exec -it movies.db /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P 'MatfRs2_Movies' -d MoviesDB -i /var/seed/seed_script.sql
```
Comment (aleksakojadinovic) [inline Services/Movies/Movies.API/dbseed.sh:1]: ovo sam zaboravio da smaknem, sklonicu

File: Services/Movies/Movies.API/Constants/ErrorMessages.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
```
Comment (aleksakojadinovic) [inline Services/Movies/Movies.API/Constants/ErrorMessages.cs:1]: i ovo isto treba da smaknem, trebalo je da ode u onu drugu granu

**Review by Hos1g4k1 — APPROVED** [review]

File: Services/Movies/Movies.API/Entities/Role.cs [hunk 3]
```diff
@@ -6,8 +6,10 @@ public class Role
     {
         [Required(ErrorMessage = "Id name is required.")]
```
Comment (VladimirV99) [inline Services/Movies/Movies.API/Entities/Role.cs:7]: Ove poruke treba da budu iz ErrorMessages

File: Services/Movies/Movies.API/Entities/Role.cs [hunk 4]
```diff
@@ -6,8 +6,10 @@ public class Role
...
         public int Id { get; set; }
+
         [Required(ErrorMessage = "Name is required.")]
         public string Name { get; set; }
+
         public string Description { get; set; }
```
Comment (VladimirV99) [inline Services/Movies/Movies.API/Entities/Role.cs:13]: Ovo polje nije oznaceno sa Required ali ne sme da bude NULL zbog tipa string

File: Services/Movies/Movies.API/Constants/ErrorMessages.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+namespace Movies.API.Constants
+{
+	public class ErrorMessages
+	{
+		public const string ID_REQUIRED = "Id is required";
+		public const string FIRST_NAME_REQUIRED = "First name is required";
+		public const string LAST_NAME_REQUIRED = "Last name is required";
+
+		public const string TITLE_REQUIRED = "Title is requiered";
```
Comment (VladimirV99) [inline Services/Movies/Movies.API/Constants/ErrorMessages.cs:10]: U ova 3 reda pise 'requiered' umesto 'required'

File: Services/Movies/Movies.API/Entities/MoviePerson.cs [hunk 6]
```diff
@@ -1,17 +1,20 @@
...
 
 namespace Movies.API.Entities
 {
     public class MoviePerson
     {
-        [Required(ErrorMessage = "MovieId is required.")]
+        [Required(ErrorMessage = ErrorMessages.MOVIE_ID_REQUIRED)]
         public int MovieId { get; set; }
-        [Required(ErrorMessage = "RoleId is required.")]
+        public Movie? Movie { get; set; }
```
Comment (VladimirV99) [inline Services/Movies/Movies.API/Entities/MoviePerson.cs:10]: Ova polja ne moraju da budu nullable, postojanje odgovarajuceg Id polja to garantuje

**Review by VladimirV99 — CHANGES_REQUESTED** [review]

**Review by VladimirV99 — APPROVED** [review]

**Pull request merged**